packages/x-components/src/views/base-config.ts

```diff
@@ -19,5 +19,6 @@ export const baseInstallXOptions: InstallXOptions = {
         identifierDetectionRegexp: '^[a-zA-Z][0-9]+'
       }
     }
-  }
+  },
+  app: {}
```

Comment:
Why do we need this empty `app` property?

Reply:
I missed it.